src/components/Sponsors/style.css

```diff
@@ -0,0 +1,54 @@
+#sponsors-wrapper {
+    margin: 20px 0;
+}
+
+#sponsors-container {
+    margin: 0 120px;
+}
+
+#sponsors, #partners {
+    font-size: 64px;
+}
+#partners {
+ margin: 20px 120px;
+}
+
+#partners-container {
```

Comment:
<img width="1103" alt="Screen Shot 2022-02-07 at 4 25 49 PM" src="https://user-images.githubusercontent.com/33165426/152894707-1b6a170e-de06-433e-8331-b710f3078afd.png">
<img width="1213" alt="Screen Shot 2022-02-07 at 4 27 26 PM" src="https://user-images.githubusercontent.com/33165426/152894908-06f6817a-5637-48a5-9d62-45892446393e.png">
We should add side margins to #partners-container and remove it from #partner-images so the images are always left aligned with the text, the same way you handled it on the sponsors container